upgrade-ops-manager/gcp/pipeline.yml

```diff
@@ -29,13 +29,27 @@ resources:
     product_version: {{opsman_major_minor_version}}
     sort_by: semver
 
+- name: tool-cliaas
```

Comment:
if this isnt already bundled into pcf-pipelines release on pivnet it should be, then we can remove this resource.